test/commons/aria/arialabel-text.js

```diff
@@ -3,32 +3,44 @@ describe('aria.arialabelText', function() {
 	var aria = axe.commons.aria;
 
 	it('returns "" if there is no aria-label', function() {
-		var node = document.createElement('div');
-		assert.equal(aria.arialabelText(node), '');
+		var vNode = new axe.SerialVirtualNode({ nodeName: 'div' });
```

Comment:
I don't think we need to specifically test `SerialVirtualNode` and instead just stick to `VirtualNode`. We already have tests for `SerialVirtualNode` that ensures it gives the correct results back to it's API, so we should just stick to testing the `arialabel-text` function and not worry about different VirtualNode types.

Reply:
VirtualNode already works. The purpose of this change is to not just ensure VirtualNode works, but that anything extending from AbstractVirtualNode works (such as Serial). VirtualNode has access to a DOM node, tests need to show it can work entirely without accessing the DOM. Passing Serial does that.

Reply:
But that would mean we duplicate every test for common functions that go though this change to ensure it works with both types. That seems like overkill and makes things much harder to maintain when we can just write a much more robust test suite for the types in one place. As long as the APIs return what we expect, we shouldn't need to write a test per type.